prboom2/data/CMakeLists.txt

```diff
@@ -362,3 +362,5 @@ add_custom_command(
     DEPENDS rdatawad ${WAD_SRC}
 )
 add_custom_target(prboomwad DEPENDS ${WAD_DATA_PATH})
+set(WAD_INSTALL_PATH ${DOOMWADDIR} CACHE PATH "Path to install prboom.wad")
```

Comment:
It's `prboom-plus.wad`, inherited mistake.

Reply:
I've made the change now to `prboom-plus.wad`. By the way, do you think it would be better for this to say "Path to install the PrBoom-Plus WAD" instead; or is the current text fine?

Reply:
I guess it's okay as it is, please push that change.